man/man1/pick.1

```diff
@@ -26,6 +26,10 @@ supports these options and arguments:
 output a help message and exit.
 .It Fl v
 output the version and exit.
+.It Fl S
+disable sorting.
+.Pp
+Instead of filtered and sorted by score, choices will only be filtered.
```

Comment:
This is missing a verb. One of these two should do it:

* Instead of being filtered and sorted by score, choices will only be filtered.
* Instead of filtering and sorting by score, only filter the choices.

Maybe we can reverse the focus:

* Only filter the choices instead of additionally sorting by score.